plugins/importexport/native/filter/NativeXmlSubmissionFileFilter.inc.php

```diff
@@ -197,10 +197,10 @@ function handleRevisionElement($node, $stageId, $fileId) {
 					$submissionFile->setUserGroupId($userGroup->getId());
```

Comment:
but the user group will be set here and after the while loop again, right? it maybe does not matter, because they should be the same...